### DISCUSSION TITLE
PP and Score ranking for each year in osu!lazer

### DISCUSSION THREAD

**Discussion by voiceofthunder0108** [ppy/osu#30585] [Ideas]
Can we introduce PP and Score ranking for each year in osu!lazer? In these rankings, only maps from a specific year would count towards pp or score, like just maps from 2007 or 2018 for example. I think this would be incredibly refreshing and interesting. We already have similar pp rankings on sites like [osuchan](https://osuchan.syrin.me/leaderboards/community/osu) but only few people participate and it would be much more appealing to have them directly in the official game. 
As an older player I’d love to come back after years, but I’d prefer not to compete in pp with Sotarks maps or other current meta maps like Sans. And I just prefer older maps over the new ones, and I know there are many players like me who feel the same way. It would also be great to have the option to display rankings by custom periods, like 2009-2014 for instance. And score ranking would finally make more sense, as there’s limited number of maps for each given year. Medals could also be given for completing or fc every map for a specific year, like 2013 completionist etc.
To summarize:
- PP ranking for each year (2007, 2008, 2009... etc.)
- Score ranking for each year
- Option to display rankings by custom periods, like 2009-2014 for instance
- Medals for fc or completing every map for a specific year
